scripts/ina260/david_new/ina260_45_short.py

```diff
@@ -0,0 +1,48 @@
+import INA260
+import RPi.GPIO as gpio
+
+pm = INA260.INA260(INA260.INA260_I2CADDR_A0A1)
+
+gpio.setmode(gpio.BCM)
+gpio.setup(17, gpio.IN)
+gpio.setup(27, gpio.IN)
+gpio.setup(22, gpio.IN)
+print(gpio.input(17))
+print("Address: 0x%X" % (pm.getAddress()))
+
+#GPI27 will go to zero during an alarm. 
+
+#print("Read Mask 0x%X" % (pm.readMaskEnable()))
+print("Alert Limit 0x%X" % (pm.readLimitRegister()))
+print(gpio.input(17))
+
+print("Initial Configuration Done\n")
+
+#pm.setUnderVoltageLimit(40.96-0.00125)
+pm.setUnderVoltageLimit(1)
+
+i = 0
+
+if pm:
+	while(i < 3): 
```

Comment:
tabs iso spaces? :cry:

Reply:
lol, it's david's script.... though I see that in all of the ina scripts now. Cleaning it up!